R/dash.R

```diff
@@ -513,21 +452,21 @@ Dash <- R6::R6Class(
         # construct function name based on package name
         fn_name <- paste0(".", pkg, "_js_metadata")
         fn_summary <- getAnywhere(fn_name)
-        
+
         # ensure that the object refers to a function,
         # and we are able to locate it somewhere
         if (length(fn_summary$where) == 0) return(NULL)
-        
+
         if (mode(fn_summary$obj[[1]]) == "function") {
           # function is available
           dep_list <- do.call(fn_summary$obj[[1]], list())
 
           return(dep_list)
         } else {
-          return(NULL)  
+          return(NULL)
         }
       })
-      
+
       deps_layout <- unlist(deps_layout, recursive=FALSE)
 
       # if core components are used, but no coreGraph() exists,
```

Comment:
Not part of this PR, but I don't think we can filter out plotly.js this way - you never know what elements will get added by a callback.

Reply:
As we discussed, this option is only supposed to be used in one case, when the app developer wishes to sidestep including a Graph component in the initial layout, but does want one following a callback:

https://github.com/plotly/dashR/blob/5f89e8034c793e2ae37266b58251887542280667/R/dash.R#L49

Reply:
Ha OK, I guess we're going to get the removal of this option from master when this PR is merged... https://github.com/plotly/dashR/commit/7663d584926ca0cb49797f7c122da9a30af5910b